apps/posts/migrations/0005_alter_post_content.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.9 on 2021-11-24 16:18
```

Comment:
we probably only need on of the migrations for posts, not sure if it will cause any problems in the future